apps/core/app/dashboard/products/_components/table/product-table-pages/product-table-pages.tsx

```
@@ -0,0 +1,53 @@
+        value={selectPage}
+        onValueChange={(value) => {
+          setSelectPage(value);
+          onChange?.(value);
+        }}
+      >
+        <SelectTrigger className="max-w-[65px]">
+          <SelectValue />
+        </SelectTrigger>
+        <SelectContent>
+          {getPages().map((number) => (
+            <SelectItem value={String(number)}>{number}</SelectItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)